src/DIRAC/MonitoringSystem/Client/DataOperationSender.py

```diff
@@ -23,7 +23,7 @@ class DataOperationSender:
 
     # Initialize the object so that the Reporters are created only once
     def __init__(self):
-        self.monitoringOption = Operations().getValue("DataManagement/MonitoringBackends", ["Accounting"])
+        self.monitoringOption = Operations().getValue("MonitoringBackends", ["Accounting"])
```

Comment:
```suggestion
        monitoringOption = Operations().getValue("MonitoringBackends", ["Accounting"])
```
No need for this to be an instance variable.

Reply:
I think it needs to since otherwise `monitoringOption` is not defined in the other methods of the class.